src/Tribe/Commerce/PayPal/Custom_Argument.php

```diff
@@ -57,6 +57,20 @@ public static function decode( $encoded, $assoc_array = false ) {
 			$encoded = str_replace( '\"', '"', $encoded );
 		}
 
+		// in case we receive the param in non json 'user_id:0,tribe_handler:tpp,pid:4' format
+		if ( ! strpos( $encoded, '{' ) && ! strpos( $encoded, '}' ) ) {
+			// we create the json from that string
+			$encoded = explode( ',', $encoded );
+
+			$encoded = array_reduce( $encoded, function ( $array, $item ) {
```

Comment:
Is this PHP 5.2 compatible?

Reply:
The callback of `array_reduce`

Reply:
It seems: http://php.net/manual/en/function.array-reduce.php

Haven't tested it, though.

Reply:
Nope, callback needs to be setup into another method.

Reply:
Woops I mean the anonymous function not `array_reduce`